wlinux-setup.d/go.sh

```diff
@@ -17,9 +17,10 @@ if (whiptail --title "GO" --yesno "Would you like to download and install the la
     export GOPATH=$HOME/go/
     export PATH=$GOPATH/bin:$GOROOT/bin:/usr/local/go/bin:$PATH
     echo "Saving Go environment variables to /etc/profile so they will persist."
-    sudo sh -c 'echo "export GOROOT=/usr/local/go" >> /etc/profile'
-    sudo sh -c 'echo "export GOPATH=\${HOME}/go/" >> /etc/profile'
-    sudo sh -c 'echo "export PATH=\${GOPATH}/bin:\${GOROOT}/bin:/usr/local/go/bin:\${PATH}" >> /etc/profile'
+    sudo sh -c 'echo "export GOROOT=/usr/local/go" >> /etc/profile.d/go.sh'
+    sudo sh -c 'echo "export GOPATH=\${HOME}/go/" >> /etc/profile.d/go.sh'
+    sudo sh -c 'echo "export PATH=\${GOPATH}/bin:\${GOROOT}/bin:/usr/local/go/bin:\${PATH}" >> /etc/profile.d/go.sh'
+    source /etc/profile.d/go.sh
```

Comment:
Looks good. I need to test this part because in my experience after the exit 0 of wlinux-setup I lose the variables. id it works there is more than a site where we can do it to avoid the user have to close and reopen WSL